backend/app/assets/javascripts/spree/backend/general_settings.js.coffee

```diff
@@ -1,8 +1,8 @@
 $(@).ready( ->
-  $('[data-hook=general_settings_clear_cache] #clear_cache').click ->
+  $('[data-hook=general_settings_clear_cache] #clear_cache').click (event) ->
     $.ajax
       type: 'POST'
-      url: '/admin/general_settings/clear_cache'
+      url: (($ event.target).attr 'data-url')
```

Comment:
Could you update this so that instead you set a route here:
https://github.com/spree/spree/blob/master/backend/app/assets/javascripts/spree/backend.js#L16-L36

Then can just call the route like: Spree.routes.clear_cache